server/service/src/processors/transfer/shipment/create_inbound_shipment.rs

```diff
@@ -145,5 +145,16 @@ fn generate_inbound_shipment(
         verified_datetime: None,
     };
 
+    system_log_entry(
+        connection,
+        match result.status.clone() {
+            InvoiceRowStatus::Picked => LogType::InvoiceStatusPicked,
+            InvoiceRowStatus::Shipped => LogType::InvoiceStatusShipped,
+            _ => LogType::InvoiceCreated,
```

Comment:
given that the fn is `generate_inbound_shipment` I wonder if this should always be `InvoiceCreated`?
I think I understand - in which case, yeah. it might be helpful to have the status of the created invoice.

Side question: is the non-exhaustive match ok? If we later add another value to the enum, then wouldn't it be better for the compiler to tell us that we need to implement a mapping? As it is, the compiler won't complain and the status will be `InvoiceCreated` which might not be correct / accurate..

Reply:
Outbound transfers should only be possible when the status is 'Picked' or 'Shipped', so the other statuses shouldn't trigger the transfer 🤞